README.md

```diff
@@ -115,11 +115,9 @@ brew install swiftgen --HEAD
 
 * Alternatively, you can clone the repository and use `rake cli:install` to build the tool and install it from any branch, which could be useful to test SwiftGen in a fork or a Pull Request branch.
 
-It is recommended to set up your Ruby environment for building using `rbenv` and `bundle`, like this.
+Some Ruby tools are used in the build process, and the system Ruby works well if you are running a recent macOS.  However, if you are a using `rbenv` you can run `rbenv install` to make sure you have a matching version of Ruby installed.  Then install the Ruby Gems:
```

Comment:
s/if you are a using/if you are using/

I'd start the "Then install the Ruby Gems" in a dedicated separate paragraph too